tools/topology/topology2/sof-ace-tplg/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ set(TPLGS
 HDA_CONFIG=mix,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,PREPROCESS_PLUGINS=nhlt,\
 NHLT_BIN=nhlt-sof-hda-generic-4ch.bin,DEEPBUFFER_FW_DMA_MS=100"
 "sof-hda-generic\;sof-hda-generic-2ch\;PLATFORM=mtl,\
-HDA_CONFIG=mix,NUM_DMICS=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-2ch.bin,\
+HDA_CONFIG=mix,NUM_DMICS=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-2ch.bin,INCLUDE_WOV=true,\
```

Comment:
That's a no for me. These topologies will be released externally, no way we are going to add a pretend WOV that makes no sense.

if you want to add WOV for tests, then that has to be done in a development topology or a CI one.

Reply:
> That's a no for me. These topologies will be released externally, no way we are going to add a pretend WOV that makes no sense.
> 
> if you want to add WOV for tests, then that has to be done in a development topology or a CI one.

@plbossart I forgot to add "Draft" on it. This is not a final solution, but a reference code. The final solution is, as Liam suggested in https://github.com/thesofproject/sof/pull/7047#pullrequestreview-1286908507, that we will create a test tplg for it. Sorry for the inconvenience.

Reply:
what you want is to extend a topology for tests not create a new one. We've done this already with deep-buffer that are not released, etc.